app/logic/serviceLearningCoursesData.py

```diff
@@ -39,6 +41,15 @@ def withdrawProposal(courseID):
     """Withdraws proposal of ID passed in. Removes foreign keys first.
     Key Dependencies: QuestionNote, CourseQuestion, CourseParticipant,
     CourseInstructor, Note"""
+
+    # delete syllabus
+    try:
+        syllabus = AttachmentUpload.select().where(AttachmentUpload.course==courseID).get()
+        FileHandler(courseId = courseID).deleteFile(syllabus.id)
+    except:
```

Comment:
We should only pass exceptions that are expected (like the file or attachment not being there), but this will pass ANY exception, including things like syntax errors